KillTeam/ViewModels/ModelsProfileViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace KillTeam.ViewModels
+{
+    public class ModelsProfileViewModel
```

Comment:
I wonder if we call this `MemberProfileViewModel` just because of `Model` and `ViewModel`?

Reply:
Models is the view it is used in.
ViewModel is it's nature.
Profile is its name ;)

Reply:
I've replied to the other comment RE: Member vs Model.